azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -1410,6 +1431,68 @@ public String handle(ResultSet rs) throws SQLException {
     }
   }
 
+  private static class FetchActiveExecutableFlowByExecId implements
+      ResultSetHandler<List<Pair<ExecutionReference, ExecutableFlow>>> {
+    private static String FETCH_ACTIVE_EXECUTABLE_FLOW_BY_EXECID =
+        "SELECT ex.exec_id exec_id, ex.enc_type enc_type, ex.flow_data flow_data, et.host host, "
+            + "et.port port, ax.update_time axUpdateTime, et.id executorId, et.active executorStatus"
+            + " FROM execution_flows ex"
+            + " INNER JOIN "
+            + " active_executing_flows ax ON ex.exec_id = ax.exec_id"
```

Comment:
Just curious: do you know which server updates table `active_executing_flows `, web server or exeuctor, or both?